forge-gui/res/cardsfolder/upcoming/trenchpost.txt

```diff
@@ -0,0 +1,8 @@
+Name:Trenchpost
+ManaCost:no cost
+Types:Land Locus
+A:AB$ Mana | Cost$ T | Produced$ C | SpellDescription$ Add {C}.
+A:AB$ Mill | Cost$ 3 T | NumCards$ X | ValidTgts$ Player | TgtPrompt$ Choose a player | AILogic$ EndOfOppTurn | SpellDescription$ Target player mills a card for each Locus you control.
+SVar:X:Count$TypeOnBattlefield.Locus+YouCtrl

```

Comment:
does this count correctly or need proper valid syntax?

Reply:
Nope doesn't work